pkg/env/env.go

```diff
@@ -0,0 +1,243 @@
+
+	// CASTemplateToDeleteCStorVolumeENVK is the ENV key that specifies the CAS Template
+	// to delete cstor cas volumes
+	CASTemplateToDeleteCStorVolumeENVK ENVKey = "OPENEBS_IO_CSTOR_CAS_TEMPLATE_TO_DELETE_VOLUME"
+
+	// CASTemplateToCreatePoolENVK is the ENV key that specifies the CAS Template
+	// to create storage pool
+	CASTemplateToCreatePoolENVK ENVKey = "OPENEBS_IO_CAS_TEMPLATE_TO_CREATE_POOL"
+
+	// CASTemplateToDeletePoolENVK is the ENV key that specifies the CAS Template
+	// to delete storage pool
+	CASTemplateToDeletePoolENVK ENVKey = "OPENEBS_IO_CAS_TEMPLATE_TO_DELETE_POOL"
+
+	// CASTemplateToListCStorSnapshotENVK is the ENV key that specifies the CAS Template
+	// to list cstor cas snapshots
+	CASTemplateToListCStorSnapshotENVK ENVKey = "OPENEBS_IO_CAS_TEMPLATE_TO_LIST_CSTOR_SNAPSHOT"
+
+	// CASTemplateToListJivaSnapshotENVK is the ENV key that specifies the CAS Template
+	// to list jiva cas snapshots
+	CASTemplateToListJivaSnapshotENVK ENVKey = "OPENEBS_IO_CAS_TEMPLATE_TO_LIST_JIVA_SNAPSHOT"
+
+	// CASTemplateToCreateJivaSnapshotENVK is the ENV key that specifies the CAS Template
+	// to create jiva cas snapshot
+	CASTemplateToCreateJivaSnapshotENVK ENVKey = "OPENEBS_IO_JIVA_CAS_TEMPLATE_TO_CREATE_SNAPSHOT"
+
+	// CASTemplateToReadJivaSnapshotENVK is the ENV key that specifies the CAS Template
+	// to read jiva cas snapshot
+	CASTemplateToReadJivaSnapshotENVK ENVKey = "OPENEBS_IO_JIVA_CAS_TEMPLATE_TO_READ_SNAPSHOT"
+
+	// CASTemplateToDeleteJivaSnapshotENVK is the ENV key that specifies the CAS Template
+	// to delete jiva cas snapshot
+	CASTemplateToDeleteJivaSnapshotENVK ENVKey = "OPENEBS_IO_JIVA_CAS_TEMPLATE_TO_DELETE_SNAPSHOT"
+
+	// CASTemplateToCreateCStorSnapshotENVK is the ENV key that specifies the CAS Template
+	// to create cstor cas snapshot
+	CASTemplateToCreateCStorSnapshotENVK ENVKey = "OPENEBS_IO_CSTOR_CAS_TEMPLATE_TO_CREATE_SNAPSHOT"
+
+	// CASTemplateToReadCStorSnapshotENVK is the ENV key that specifies the CAS Template
+	// to read cstor cas snapshot
+	CASTemplateToReadCStorSnapshotENVK ENVKey = "OPENEBS_IO_CSTOR_CAS_TEMPLATE_TO_READ_SNAPSHOT"
+
+	// CASTemplateToDeleteCStorSnapshotENVK is the ENV key that specifies the CAS Template
+	// to delete cstor cas snapshot
+	CASTemplateToDeleteCStorSnapshotENVK ENVKey = "OPENEBS_IO_CSTOR_CAS_TEMPLATE_TO_DELETE_SNAPSHOT"
+
+	// CASTemplateToReadVolumeStatsENVK is the ENV key that specifies the CAS Template
+	// to read volume stats
+	CASTemplateToReadVolumeStatsENVK ENVKey = "OPENEBS_IO_CAS_TEMPLATE_TO_READ_CAS_VOLUME_STATS"
+
+	// CASTemplateToListStoragePoolENVK is the ENV key that specifies the CAS Template
+	// to list cas sto
+	CASTemplateToListStoragePoolENVK ENVKey = "OPENEBS_IO_CAS_TEMPLATE_TO_LIST_STORAGE_POOL"
+
+	// CASTemplateToReadStoragePoolENVK is the ENV key that specifies the CAS Template
+	// to read storagepool
+	CASTemplateToReadStoragePoolENVK ENVKey = "OPENEBS_IO_CAS_TEMPLATE_TO_READ_STORAGE_POOL"
+)
```

Comment:
nit: I think we don't need all these env's

Reply:
removed